File: trunk/C#/PokerProtocol/Commands/Lobby/AbstractLobbyResponse.cs
using System;
using System.Collections.Generic;
using System.Text;
using EricUtility;
using EricUtility.Networking.Commands;

namespace PokerProtocol.Commands.Lobby
{
    public abstract class AbstractLobbyResponse<T>: AbstractTextCommandResponse<T>
        where T : AbstractLobbyCommand
    {
        protected override void Append<T2>(StringBuilder sb, T2 thing)
        {
            sb.Append(thing);
            sb.Append(AbstractLobbyCommand.Delimitter);
        }

        public AbstractLobbyResponse(T command) : base(command)
        {
        }

    }
}

File: trunk/JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands/lobby/AbstractLobbyResponse.java
package bluffinmuffin.protocol.commands.lobby;

import bluffinmuffin.protocol.commands.AbstractCommandResponse;

public abstract class AbstractLobbyResponse<T extends AbstractLobbyCommand> extends AbstractCommandResponse<T>
{
    
    public AbstractLobbyResponse(T command)
    {
        super(command);
    }
    
    @Override
    protected <T2> void append(StringBuilder sb, T2 thing)
    {
        sb.append(thing);
        sb.append(AbstractLobbyCommand.Delimitter);
    }
    
}
